imagenet/model_saver.py

```diff
@@ -10,9 +10,12 @@ def save_model(device: str = "CPU"):
             raise Exception("Requested to create GPU model, but CUDA was not found.")
     # Device's torch name
     device_name = "cuda" if device.startswith("GPU") else "cpu"
-    # model = models.resnet50(pretrained=True)
+
+    # Many of the Imagenet models could be used. shufflenet has about the the same number of
+    # parameters as the ones used in our use cases
     model = models.shufflenet_v2_x0_5(pretrained=True)
-    #model = models.resnet152(pretrained=True)
+    # model = models.resnet50(pretrained=True)  # ~250k parameters
```

Comment:
hmmm I think the commented out models.resnet50 code is for users to uncomment -> possibly add this somewhere in the documentation that you can navigate to model_saver.py and uncomment to change

Possibly here https://github.com/CrayLabs/SmartSim-Scaling/tree/master/driverinference under supported inference tests? 

or we can create a ticket bc is it bad that we make the user uncomment code?

Reply:
Modified this to allow the user to choose which of the torchvision models they wanted